src/cli.js

```diff
@@ -0,0 +1,6 @@
+const maven = requir('./index');
```

Comment:
Hi @reflexdemon,

I think there is a typo `requir` is missing an `e` in the end.

Reply:
That is a good catch. I will revisit and update shortly

Reply:
Actually that file was not required. It is never used inside the application and I have removed the file now. Thanks for bring this to my attention.